shared/fs/filepreview/text-view.js

```diff
@@ -17,9 +20,26 @@ const TextView = (props: TextViewProps) => (
 // We need to do the spacing in the guest content of the webView rather than
 // the component's styles, to make it feel like the whole "view" is
 // scrollable".  The <body> element has the actual content, while <html>
-// provides the top and bottom margin that blends with the left and right space
-// provided by <FilePreview/>
-const webviewCSS = `
+// provides the top and bottom margin that blends with the rest of the app.
+const webviewCSS = Styles.isMobile
+  ? `
+html{
+  background-color: ${Styles.globalColors.blue5};
+  padding-top: ${Styles.globalMargins.mediumLarge};
+  padding-bottom: ${Styles.globalMargins.mediumLarge}; 
+  margin: 0;
+}
+body{
+  background-color: ${Styles.globalColors.white};
+  padding: ${Styles.globalMargins.medium};
+  margin: 0;
+  color: ${Styles.globalColors.black};
+  font-size: 15;
+  line-height: 1.6;
+  font-family: ${Styles.globalStyles.fontTerminal.fontFamily}
```

Comment:
I'm realizing this is my fault, but:
1. This needs to be in quotes, and needs to have a semicolon.

Reply:
Good catch! While playing with that, I realized this is actually ineffective on `body` since there's an automatic `pre` style inserted that sets it to `monospace` and overrides what's from `body`. So I've also moved `font-family` into an explicit `pre` block.